pkg/interactive/bubbletea/machinepoolbubble/transcript.go

```
@@ -0,0 +1,86 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package machinepoolbubble
+
+import (
+	"fmt"
+
+	"github.com/charmbracelet/lipgloss"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)